### DISCUSSION TITLE
full or partial html support

### DISCUSSION THREAD

**Discussion by lakhininenina** [misskey-dev/misskey#8743] [Q&A]
it seems html is supported. there are couple of issues which state html can be converted to mfm.
do u support full or partial html in notes or pages or posts?

**Comment by ThatOneCalculator** [comment]
The only "html-like" tags are `i` for italic, `b` for bold, `small`, and `center`, which are all part of Misskey Flavored Markdown (MFM)

**Reply by lakhininenina** [reply]
interesting. i am unable to find docs in eng, but japanese version is promising https://github.com/misskey-dev/mfm.js/blob/c2f166d4668f76b411749bc35e65cc5c8185f635/docs/syntax.md

**Comment by Johann150** [comment]
Misskey does not currently support HTML input. HTML to MFM conversion happens for content that comes in via federation from instances that do not run Misskey, since HTML seems to be a well established markup format in the Fediverse.

I think the most fitting currently open issue is https://github.com/misskey-dev/misskey/issues/4244.
